web/templates/topic_sidebar_actions.hbs

```diff
@@ -76,6 +76,18 @@
             {{t "Move topic"}}
         </a>
     </li>
+    {{else if can_rename_topic}}
+    <hr />
+
+    <li>
+        <a tabindex="0" class="sidebar-popover-rename-topic-messages" data-stream-id="{{ stream_id }}" data-topic-name="{{ topic_name }}">
+            <i class="fa fa-pencil" aria-hidden="true"></i>
+            {{t "Rename topic"}}
+        </a>
+    </li>
+    {{/if}}
+
+    {{#if can_move_topic}}
```

Comment:
I guess as a follow-up we should probably make this be controlled by a separate `settings_data` query; which I think should for now its value be equal to `can_rename_topic`, since resolving topics certainly doesn't require permission to move things between streams.

Reply:
Sounds good.